### PR TITLE
Carousel and eye

### PR DESCRIPTION
**Pull request by Quentin-Macquart**

### CODE REVIEW

**Review by BBriset — APPROVED** [review]
looks ok

File: src/components/carousel/carousel.css [hunk 1]
```diff
@@ -1,11 +1,45 @@
...
+  }
+
+  .imgContain {
+    display: flex;
+    justify-content: center;
+    margin-top: 10vh;
+  }
+}
+
+/* .imgContain:hover {
```
Comment (Hacene010) [inline src/components/carousel/carousel.css:38]: Dead code !!!!

File: src/components/eye/Eye.jsx [hunk 2]
```diff
@@ -6,7 +6,9 @@ const Eye = () => {
     <>
       <main className="card-container">
```
Comment (Hacene010) [inline src/components/eye/Eye.jsx:7]: Try to use camelCase for classNames everywhere to be coherent.

**Review by Hacene010 — APPROVED** [review]
Seems ok for me

**Pull request merged**